services/motion/server_test.go

```diff
@@ -137,6 +142,561 @@ func TestServerMoveOnGlobe(t *testing.T) {
 	test.That(t, moveOnGlobeResponse.GetSuccess(), test.ShouldBeFalse)
 }
 
+func TestServerMoveOnGlobeNew(t *testing.T) {
```

Comment:
I added tests for all the new methods as the server has some logic about how to handle inputs to & outputs from the go interface method & that logic wasn't tested anywhere else.